Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "node-bindgen"
-version = "6.0.2"
+version = "6.0.3"
```

Comment:
Can you change to "6.1.0" since its more than patch?

Reply:
@sehz done